safesocket/unixsocket.go

```diff
@@ -64,7 +64,7 @@ func listen(path string, port uint16) (ln net.Listener, _ uint16, err error) {
 	if err != nil {
 		return nil, 0, err
 	}
-	os.Chmod(path, 0600)
+	os.Chmod(path, 0666)
```

Comment:
Given we're only using SO_PEERCRED on linux, use a restricted mask on other Unixes?

Reply:
SGTM. I'd want to unify the knowledge of able-to-ucred in one place so these don't get out of sync.